environment.yml

```
@@ -6,7 +6,7 @@ channels:
 dependencies:
 - python=3.6
 - numpy
-- scipy
+- scipy>=1.4.0
 - matplotlib
```

```suggestion
- matplotlib
- cython
```

Add cython directly ?

---

I was hesitant to do this because I wasn't sure if installing the cython module also installs a c-compiler. I suppose we can give it a go and see if it causes problems?

---

Yes ! We already have Numba as a requirement (for Whiting's) so there must be some compilers required there too !